src/screens/SUE/SueReportScreen.tsx

```diff
@@ -650,5 +687,8 @@ const styles = StyleSheet.create({
   },
   contentContainer: {
     width: device.width
+  },
+  noPaddingTop: {
+    paddingTop: 0
```

Comment:
and regarding the divider with open description, it should be full width to correctly divide the screen parts. left and right parts are missing of the divider line:

<img width="401" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/a216eb13-55e2-48af-bde8-e4f58976b15a">

Reply:
divider updated to the width of the screen 👍